### PR TITLE
Fixes #20665: Demote relay rudder server command

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/20665

### CODE REVIEW

**Comment by ElaadF** [comment]
Commit modified

**Comment by ElaadF** [comment]
Commit modified

**Comment by ElaadF** [comment]
Commit modified

**Comment by ElaadF** [comment]
Commit modified

**Comment by ElaadF** [comment]
Commit modified

**Comment by ElaadF** [comment]
Commit modified

File: scale-out-relay/src/main/resources/server-relay-to-node [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+if [ "${NODE_UUID}" = "" ]; then
+	echo "ERROR: Need the UUID of the Node as an argument, aborting."
+	usage
+	exit 1
+elif [ "${NODE_UUID}" = "aaaaaaaa-bbbb-cccc-dddd-eeeeeeee" ]; then
+	echo "ERROR: Please use a real UUID, not the one supplied in the documentation, aborting."
+	exit 1
+fi
+
+echo "Rudder relay installation script starting, using UUID ${NODE_UUID}."
```
Comment (amousset) [inline scale-out-relay/src/main/resources/server-relay-to-node:54]: shouldn't be here

**Comment by ElaadF** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Pull request merged**